kola/tests/coretest/core.go

```diff
@@ -214,6 +215,26 @@ func TestReadOnlyFs() error {
 	return fmt.Errorf("could not find /usr or / mount points.")
 }
 
+// Test that the root disk's UUID was set to a random one on first boot.
+func TestFsRandomUUID() error {
+	var out bytes.Buffer
+	c := exec.Command("sh", "-ec", "sudo blkid -o value -s PTUUID /dev/$(lsblk -no PKNAME $(findmnt -vno SOURCE /))")
+	c.Stdout = &out
+	err := c.Run()
+	if err != nil {
+		return fmt.Errorf("findmnt: %v", err)
+	}
+
+	defaultUUID := []byte("00000000-0000-0000-0000-000000000001")
+	got := bytes.TrimSpace(out.Bytes())
```

Comment:
perhaps use the uuid library to actually parse the output so we know it dodn't fail in a weird way.